mifospay/src/main/AndroidManifest.xml

```diff
@@ -17,16 +17,20 @@
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
         <activity android:name=".merchants.ui.MerchantTransferActivity"></activity>
-        <activity
-            android:name=".auth.ui.LoginActivity"
-            android:theme="@style/AppTheme"
-            android:windowSoftInputMode="stateVisible">
+
+	<activity
+            android:name=".SplashScreenActivity"
+            android:theme="@style/SplashTheme">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
-
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        
+	<activity
+            android:name=".auth.ui.LoginActivity"
+            android:theme="@style/AppTheme"
```

Comment:
@devansh-299 Just popped out of my mind, did you have to add the entire LoginActivity again. I remember we having it. Perhaps the change only is enough. I might be wrong as well. Please correct me if I am :)

Reply:
@Bhavnaharitsa I didn't add `LoginActivity` again, I just removed Launcher filter from this activity and used `SplashScreenActivity` as the new Launcher activity.

Reply:
Yess, I agree. But the commit message shows as such :)

Reply:
> Yess, I agree. But the commit message shows as such :)

yeah, the Github's change tracker is a little bit misleading

Reply:
Haha, same happened with me in the previous pr GitHub is surely misleading